### ISSUE TITLE
mobile: p:calendar loses styling on ajax update

### ISSUE THREAD

**Issue by fleske** [primefaces/primefaces#868]
Hello, i have the same issue described like these two described a long time ago - but the problem still exists in PF 5.3. Any help please?

https://code.google.com/p/primefaces/issues/detail?id=7135
http://forum.primefaces.org/viewtopic.php?f=8&t=38817

Thanks

**Comment by ciroppina** [comment]
Same problem with pf 5.3.8 (18-Oct-2015) and 
- JSF jboss-jsf-api_2.2_spec
- javax el-api 2.2.1-b04
- el-impl .0

deploying and running on:
- JBoss EAp 6.1 (7.2.0 Final)

Here the code:

XHTML page:

```
<f:view renderKitId="PRIMEFACES_MOBILE" />
...
<h:body>

        <pm:page>
...
<p:calendar
    id="startDate"
    immediate="true"
    mode="popup"
    navigator="true"
    value="#{startAppModel.aSQLDate}"
    valueChangeListener="#{startAppModel.handleCalendarChanged()}"
    yearRange="c-1:c+1"
    style="width: 100px; padding: 10px 5px;"
    size="7"
    pattern="yyyy-MM-dd"
    readonlyInput="true"
    locale="it"
>
</p:calendar>
</h:form>
```

My Java @Model

```
@Model
@SessionScoped // javax.enterprise.context
public class StartAppModel implements Serializable {

    private static final long serialVersionUID = -7461606818066615154L;

    @Inject
    ExchangeWSApp app;

    private Date aSQLDate;
    public Date getaSQLDate() {
        return aSQLDate;
    }
    public void setaSQLDate(Date aSQLDate) {
        this.aSQLDate = aSQLDate;
    }
...
```

**Comment by tandraschko** [comment]
I check this some months ago but AFAIR this was not fixable on our side and is a bug in jquery mobile.

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
